packages/compass-connections/src/stores/connections-store-redux.ts

```diff
@@ -1509,10 +1509,6 @@ export const connect = (
       track(
         'Connection Attempt',
         {
-          is_favorite: connectionInfo.savedConnectionType === 'favorite',
```

Comment:
drive by, these attributes aren't meaningful anymore with multiple connections

Reply:
Are they, though? We still have the concept of a favourite that we haven't removed it. Can't ever remove it if we don't keep stats on whether it gets used.

Reply:
[Relevant thread](https://mongodb.slack.com/archives/G2L10JAV7/p1724951574301919)

I suggested removing them because the significance of is_favorite is not the same as it was before. Earlier is_favorite was the only way to differentiate wether a connection is saved or recent but now all the connections are saved and is_favorite is more UI related, allowing us to group connections together. So for the telemetry point that it used to represent earlier, it does not make sense to keep it anymore. However if we would like to track connections being marked as favorite maybe we can introduce a different attribute?

Reply:
ahh ok, so is favorite is still a thing, then i will keep it

Reply:
and is recent doesn't make sense because everything saved now right?

Reply:
I think is_recent can be dropped, yes.

Reply:
ok re-added is_favorite then! thanks!